packages/jwt-verifier/tools/runIntegrationTests.sh

```diff
@@ -1,11 +1,4 @@
-
 #!/bin/bash
 . "../../scripts/tck.sh"
 
 runTest "./test/integration-test/resources/testRunner.yml" "target/cli-test-output" "./test/integration-test/resources/testng.xml"
```

Comment:
Whats stopping us from moving this inside of `tck.sh` too? That way we can run the following to our `package.json`:

```json
"scripts" : {
  "test:integration": "bin/bash ../../scripts/tck.sh"
}
```

We'd be passing in our current process path, so we should be able to infer what package triggered the tests.

Reply:
Nothing is stopping us for now.
I was wondering if we need to add additional arguments to run TCK tests in the future, we will have to break them again. Not likely to happen but still...
Also, having the scripts that run the tests in their own repo would be nice in terms of readability?

Reply:
Yeah - there are a couple of ways we could do this, but we can explore this at a later point in time.

For now, I'd like us to focus on reducing the complexity of one script calling another script.